error-prone-contrib/src/main/java/tech/picnic/errorprone/refasterrules/StreamRules.java

```diff
@@ -168,9 +168,8 @@ Stream<R> after(Stream<T> stream, Function<? super S, ? extends Stream<? extends
    * Where possible, clarify that a mapping operation will be applied only to a single stream
    * element.
    */
-  // XXX: Implement a similar rule for `.findAny()`. For parallel streams this wouldn't be quite the
-  // same, so such a rule requires a `Matcher` that heuristically identifies `Stream` expressions
-  // with deterministic order.
+  // XXX: Consider whether to have a similar rule for `.findAny()`. For parallel streams it
+  // wouldn't be quite the same...
```

Comment:
That's the last bit of unintended changes in this PR. :eyes:
See https://github.com/PicnicSupermarket/error-prone-support/pull/327/files#diff-35c5c43be02c59bf4ed5296465d36cc5adc02addcf0269f7264332cde0f0d779R171-R173

Reply:
Tnx!